Dockerfile

```diff
@@ -1,7 +1,6 @@
+# -*- mode: dockerfile-mode; docker-image-name: "tailscale-dev"; -*-
 # Install dependencies only when needed
 FROM node:18-alpine AS builder
-# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
-RUN apk add --no-cache libc6-compat
 WORKDIR /app
 COPY . .
 RUN yarn install --frozen-lockfile
```

Comment:
I wasn't able to get `./tool/yarn` working in Alpine due to ABI conflicts because Node is written in C++ and the musl C++ ABI is just slightly different enough that gcompat can't figure it out.

Reply:
File an issue?

Reply:
https://github.com/tailscale-dev/tailscale-dev/issues/55